### PR TITLE
Fullscreen support

### LINKED ISSUE #63: [feat/fullscreen] Fullscreen functionality with custom button click on video player
Is There a way to call a fullscreen functionality with custom button (Not default video player one button). 
Right now i have tried to build some customization on [feat/fullscreen](https://github.com/bitmovin/bitmovin-player-react-native/tree/feat/fullscreen) branch but getting this error.
Screenshot 2022-12-14 at 8 51 40 PM
Added this code in [RNPlayerViewManager](https://github.com/bitmovin/bitmovin-player-react-native/blob/feat/fullscreen/ios/RNPlayerViewManager.swift).
```
@objc func goToFullscreen(_ viewId: NSNumber, playerId: NativeId) {
        bridge.uiManager.addUIBlock { [weak self] _, views in
            guard
                let view = views?[viewId] as? RNPlayerView,
                let player = self?.getPlayerModule()?.retrieve(playerId)
            else {
                return
            }
            view.onFullscreenRequested()
        }
    }
```

        `@objc func exitFullscreen(_ viewId: NSNumber, playerId: NativeId) {
        bridge.uiManager.addUIBlock { [weak self] _, views in
            guard
                let view = views?[viewId] as? RNPlayerView,
                let player = self?.getPlayerModule()?.retrieve(playerId)
            else {
                return
            }
            view.onFullscreenExitRequested()
        }
    }`

### PR DESCRIPTION
**Pull request by rolandkakonyi**
close #63

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ Official React Native bindings for Bitmovin's mobile Player SDKs.
       - [Prepare hooks](#prepare-hooks)
     - [Adding external subtitle tracks](#adding-external-subtitle-tracks)
     - [Enabling Picture in Picture mode](#enabling-picture-in-picture-mode)
+    - [Setting up fullscreen handling](#setting-up-fullscreen-handling)
```
Comment (rolandkakonyi) [inline README.md:36]: @bitbalint please review the README changes

File: example/src/screens/BasicFullscreenHandling.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+          error
+        );
+      });
+
+      // Load desired source configuration
+      player.load({
+        url:
+          Platform.OS === 'ios'
+            ? 'https://bitmovin-a.akamaihd.net/content/MI201109210084_1/m3u8s/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.m3u8'
+            : 'https://bitmovin-a.akamaihd.net/content/MI201109210084_1/mpds/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.mpd',
```
Comment (strangesource) [inline example/src/screens/BasicFullscreenHandling.tsx:77]: Why don't we use the hls source for Android? just curious.

**Review by strangesource — APPROVED** [review]
Looks good to me, just some NIT.

File: example/src/screens/BasicFullscreenHandling.tsx (hunk 2 above)
Comment (rolandkakonyi) [reply example/src/screens/BasicFullscreenHandling.tsx:77]: We do this all the time, maybe to showcase we can do DASH on Android.

**Review by matamegger — APPROVED** [review]
Nice 👍

**Pull request merged**